blog/2023-10-02-network.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Network Team Update
+slug: 2023-10-02-network
+authors: coot
+tags: [network]
+hide_table_of_contents: false
+---
+
+## High-level overview of [sprint 45][sprint-45]
+
+[sprint-45]: https://github.com/orgs/input-output-hk/projects/19/views/16?filterQuery=sprint%3A%22sprint+45%22
+
+### Bootstrap Peers
+
+We started reviewing the bootstrap peers PR, [ouroboros-network#4615].
+
+### Towards Typed Protocols 0.2.0.0
+
+We discovered a performance regression when using `typed-protocols-0.2.0.0`, and we
+started investigating where it comes from.  Currently, we see that
+`typed-protocols-0.2.0.0` can outperform `typed-protocols-0.1.0.0` when running
+in isolation with a simple `ping-pong` protocol, so the regression might be in
+the new block fetch implementation which comes with `typed-protocols-0.2.0.0`
+See 
+
+### Tech Debt
+
+We merged two PRs written by Galois engineers:
+
+* a pull request which refactors the main entry function for P2P, see [ouroboros-network#3834]; 
+* a pull request which reviews usage of `unsafe` function in the network code based.
+
+### IO-Sim
+
+#### IOSimPOR
+We found and fixed a bug in `IOSimPOR`.  We'd like to thank Prof. John Hughes
+(Quviq AB) for helping us with debugging the issue.
```

Comment:
I think Quviq is misspelled.

Reply:
I think it's right, I don't see difference from how you spelled it.